.github/workflows/project_management.yml

```diff
@@ -0,0 +1,16 @@
+name: project-management
+
+on:
+  issues:
+    types: [opened]
```

Comment:
This will work great, but if ever move an issue back to triage, the automation won't pick it up (unless we use the labeled event).

Reply:
I think that's ok for now. I want to see how often an issue ends up needing to be re-triaged.